Configuration/StandardSequences/python/RawToDigi_Data_cff.py

```diff
@@ -6,3 +6,10 @@
 ecalDigis.DoRegional = False
 #False by default ecalDigis.DoRegional = False
 
+# RPC Merged Digis 
+_muonRPCDigis = muonRPCDigis.copy()
+from Configuration.Eras.Modifier_run3_RPC_cff import run3_RPC
+_muonRPCDigis.inputTagTwinMuxDigis = cms.InputTag('rpcTwinMuxRawToDigi')
+_muonRPCDigis.inputTagOMTFDigis = cms.InputTag('omtfStage2Digis')
+_muonRPCDigis.inputTagCPPFDigis = cms.InputTag('rpcCPPFRawToDigi')
+run3_RPC.toReplaceWith(muonRPCDigis,_muonRPCDigis)
```

Comment:
Why not
```python
run3_RPC.toModify(muonRPCDigis,
    inputTagTwinMuxDigis = 'rpcTwinMuxRawToDigi',
    inputTagOMTFDigis = 'omtfStage2Digis',
    inputTagCPPFDigis = 'rpcCPPFRawToDigi'
)
```
?

Reply:
Yes. Anything else or can I push this modification?